js/privatebin.js

```diff
@@ -3313,7 +3342,7 @@ jQuery.PrivateBin = (function($, RawDeflate) {
          */
         me.addComment = function(comment, commentText, nickname)
         {
-            if (commentText === '') {
+            if (!commentText) {
```

Comment:
I know this works, but the `=== ''` is more readable and therefore the intent is more obvious. The `!` is easy to miss.

Reply:
Ok, fair enough, the `not` will catch that, while the comparison with empty string will not. It is probably a bug in itself, but your fix catches it for now.